SS_Activation_OpenMC.py

```diff
@@ -127,25 +127,29 @@
 # Stable W nuclides present at beginning of operation (will not be plotted)
 stable_nuc = ['W180', 'W182', 'W183', 'W184', 'W186']  
 
-#Store list of nuclides from last timestep as a Materials object
-materials_last = results.export_to_materials(-1)
-# Storing depletion data from 1st material
-mat_dep = materials_last[0]
-# Obtaining the list of nuclides from the results file
-nuc_last = mat_dep.get_nuclides()
+materials_list=[]
```

Comment:
These changes don't seem related to this PR.  Maybe you branched from another branch, but if you are only changing the ALARA inputs in ways that don't depend on these changes, you should branch from main and make a self-contained PR for just those things.